### DISCUSSION TITLE
Save not working or what?

### DISCUSSION THREAD

**Discussion by Dan-Essin** [gildas-lormeau/SingleFile#1009] [General]
I normally save all my downloads and captured pages to a samba server - not on  the local machine.

Using Fedora 36 with Firefox.
FF configured to prompt for download save location.
"Saved" single page.
Where did it go? It's not in the Downloads folder.

If I knew the file-naming convention, I could probably search the entire machine for it, but that's really not worth the effort. The File|Save Page... is only an extra click away.

There needs to be an option to prompt for the save-to location on every invocation. Otherwise, not very useful but has potential for me.

Hope that you add this feature.

**Comment by gildas-lormeau** [comment]
Please check the option `File name > open the "Save as" dialog to confirm the file name` in the options page of SingleFile.
